topics/ecology/tutorials/Biodiversity data exploration/tutorial.md

```diff
@@ -40,255 +46,231 @@ This tutorial will guide you on the exploration of biodiversity data having taxo
+>    for you to find it again later if needed.
 > 2. Import the files from [Zenodo](https://zenodo.org/record/5930763/files/Reel_life_survey_fish_sample.tabular?download=1)
 >
 >    {% snippet faqs/galaxy/datasets_import_via_link.md %}
 >
 >    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
 >
-> 3. Rename the datasets "reef_life_fish" for example
-> 4. Check that the datatype is tabular
+> 3. Rename the datasets "reef_life_molluscs" for example and preview your dataset
+>
+>    You can see that the dataset hasn't been detected to be a CSV dataframe, it is because RLS data directly puts the
+>    metadata of the dataframe in the first lines before the dataframe so you'll have to remove these lines using the {% tool [Remove beginning](https://toolshed.g2.bx.psu.edu/repos/devteam/remove_beginning) %} with the following parameters:
+>        - {% icon param-text %} *"Remove first"*: `72`
+>        - {% icon param-file %} *"from"*: reef_life_molluscs data file
 >
->    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>    Then, verify if your new file hasn't got hashtags in the first lines and modify the datatype of the new file from txt to csv.
+>    {% include snippets/change_datatype.md datatype="csv" %}
 >
-> 5. Add to each database a tag corresponding to ...
+> 4. Convert datatype CSV to tabular 
 >
->    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>    {% include snippets/convert_datatype.md conversion="Convert CSV to tabular" %}
 >
 {: .hands_on}
 
-# Want to spatially anoymize your data 
-
-A first step of this tutorial will show you how you can simply apply spatial coordinates anonymization if you want to share data without spatial context.
-
+## Prepare the data
 
-## Sub-step with **Spatial coordinates anonymization**
+In order to clean unnecessary informations from the table we will now cut a few columns and change the format of time information.
 
-> ### {% icon hands_on %} Hands-on: Task description
+> ### {% icon hands_on %} Hands-on: Clean your data
 >
-> 1. {% tool [Spatial coordinates anonymization](toolshed.g2.bx.psu.edu/repos/ecology/tool_anonymization/tool_anonymization/0.0.0) %} with the following parameters:
->    - {% icon param-file %} *"Input table"*: `output` (Input dataset)
->    - *"Select column containing latitudes in decimal degrees"*: `c9`
->    - *"Select column containing longitudes in decimal degrees"*: `c10`
+> 1. Use {% tool [Cut columns from a table (cut)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cut_tool/1.1.0) %}
+>    with following parameters :
+>    - {% icon param-files %} *"File to cut"*: Convert CSV to tabular data file
+>    - {% icon param-select %} *"Operation"*: `Keep`
+>    - {% icon param-select %} *"Delimited by"*: `Tab`
+>    - {% icon param-select %} *"Cut by"*: `fields`
+>      - {% icon param-select %} *"List of Fields"*: `Column: 8` `Column: 10` `Column: 11` `Column: 12` `Column: 25` `Column: 28`
```

Comment:
Add the real header names of the columns and justify that we just keep the columns we need for the tutorial but it doesn't mean the other columns aren't interesting